pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -409,26 +411,78 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			Fn:           flow.TaskFn(botanist.InitializeDesiredShootClients).RetryUntilTimeout(defaultInterval, 2*time.Minute),
 			Dependencies: flow.NewTaskIDs(waitUntilKubeAPIServerIsReady, waitUntilControlPlaneExposureReady, waitUntilControlPlaneExposureDeleted, deployInternalDomainDNSRecord, deployGardenerAccess),
 		})
-		rewriteSecretsAddLabel = g.Add(flow.Task{
-			Name: "Labeling secrets to encrypt them with new ETCD encryption key",
+		rewriteResourcesAfterEncryption = g.Add(flow.Task{
```

Comment:
"AfterEncryption" sounds misleading. Nothing is encrypted yet. Rather the config changed.